packages-ts/gauntlet-terra-contracts/src/commands/contracts/multisig/wallet.ts

```diff
@@ -0,0 +1,60 @@
+import { CATEGORIES } from '../../../lib/constants'
+import { AbstractInstruction, instructionToCommand } from '../../abstract/wrapper'
+import { Duration, isValidDuration, Threshold, isValidThreshold } from './lib/types'
+import { isValidAddress } from '../../../lib/schema'
+
+type WalletInitParams = {
```

Comment:
Let's clearly differentiate the command input vs the contract input in every command for consistency. Command input should always be on camelCase.
`WalletInitParams` is the contract input, but not necessarily the command